russian/vesti-nedeli/vesti.md

```diff
@@ -0,0 +1,5 @@
+Вести вселяют смысл. Зефирная комната, подмышечный оркестр, ящик водки - грани единого плана, несмотря на неясные очертания,План несомненно велик.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.